src/airport.js

```diff
@@ -1,10 +1,15 @@
 class Airport {
   planeList = [];
-  capacity = 7;
 
-  landPlane = plane => { this.planeList.push(plane) };
+  constructor(capacity = 7) {
+    this.capacity = capacity;
+  };
+
+  addPlane = plane => { if (!this.airportFull()) this.planeList.push(plane) };
```

Comment:
I change the name of this a few times and it was a great source of confusion while writing tests. Next time I plan on spending more time thinking about clear, descriptive and useful function and variable names before I start coding